src/app/admin/pages/learning-objects/learning-objects.component.ts

```diff
@@ -64,6 +64,8 @@ export class LearningObjectsComponent
 
   isAdminOrEditor: boolean;
 
+  isCurator: boolean;
+
```

Comment:
This doesn't look like it's being used. Please remove if it isn't.

Reply:
I believe it being initialized in line [131](https://github.com/Cyber4All/clark-client/pull/1602/files#diff-2fd4bd2324265dd5e8d85292421d4c9d9f6107cf48f47106b768e7e06c6aab28R131) to a boolean value to check if the current user has a curator access.